wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/wsdfu/DFUFileAccessInfoWrapper.java

```diff
@@ -143,13 +143,6 @@ public DFUFileAccessInfoWrapper(org.hpccsystems.ws.client.gen.axis2.wsdfu.v1_39.
         }
 
         int dataFilePartsCount = fileParts.length;
-        if (fileType == DFUFileTypeWrapper.Index && dataFilePartsCount % availableLocations.size() != 0) // apparently not all keyed files report TLK
```

Comment:
per offline conversation, we might want to mask the TLK file part if HPCCFile will automatically consume it